src/events/dns.js

```diff
@@ -170,10 +143,10 @@ function wrapDnsResolveFunction(original) {
         if (typeof arg2 === 'object') {
             options = arg2;
         }
-        const { dnsEvent, startTime } = initialDnsEvent(original.name);
+        const { event, startTime } = initializeEvent('dns', original.name, 'dns');
```

Comment:
I would still use `event: dnsEvent` - the word event is too generic and reading the code might become confusing